### DISCUSSION TITLE
Running verdaccio inside Kuberentes with an ALB

### DISCUSSION THREAD

**Discussion by GregSharpe1** [verdaccio/verdaccio#3178] [General]
Hey all, is anyone running Verdaccio inside kubernetes with an ALB? I'm in the process of moving verdaccio from ELBv1 over to ELBv2 (ALB) and a very noticeable latency issue. I was just wondering if anyone else has been having a similar problem?

**Comment by juanpicado** [comment]
@verdaccio/kubernetes maybe <-- they know

**Comment by kav** [comment]
How are you integrating the ALB? Using the load balancer controller and via the ingress in the chart?

**Reply by GregSharpe1** [reply]
Yes exactly that, using an ACM cert which is automatically picked up by the controller too if that helps :)

**Comment by pshanoop** [comment]
I was using it with ALB, We didn't have any latency difference. 

How did you find out it's the ALB related issue? 
Is the latency lower on if the pod [port-forward](https://kubernetes.io/docs/tasks/access-application-cluster/port-forward-access-application-cluster/) used.
